corehq/apps/user_importer/models.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+from csv import DictWriter
+from io import StringIO
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from corehq.blobs import CODES, get_blob_db
+
+
+class UserUploadRecord(models.Model):
+    domain = models.TextField()
```

Comment:
Why is this a `TextField` instead of a `CharField`?

Reply:
On the backend its all the same in postgres (char columns are just text columns with a length constraint), so I tend to use TextField for everything unless there is a good reason to include a limit, which I think is a convention we have. If part of that is why not a limit, it didn't seem to buy us anything since we want to include whatever the domain name is, so we would not actually be using this column to limit domain name.

Reply:
For domain columns we use `CharField` almost always, although the character limits vary widely (@kaapstorm didn't you graph them recently?). But if they're the same underneath, `TextField` sounds fine. I'm in favor of constraining columns as much as possible - length limits, disallowing null, whatever can be restricted - on the principle that it's always easier to loosen constraints than to tighten them.

Reply:
I'd favour CharField even if it does not make any difference performance/storage wise because
1. that way it would be consistent with how we store "domain" on other tables.
2. for postgres these two would be the same, but generally independent of backend, I'd understand a text field as something that is a longer text/para whereas this is just a word.